src/translations/v6/numeric-big-integer-hack.ts

```diff
@@ -0,0 +1,34 @@
+import SbvrTypes from '@balena/sbvr-types';
+import type * as TypeUtils from '@balena/sbvr-types/out/type-utils.js';
+
+// We use the TsTypes as the type name here so that it doesn't conflict later
+// when we augment the @balena/sbvr-types Types type.
+type TsTypes = TypeUtils.TsTypes<number, number>;
+export { TsTypes as Types };
+export type DbWriteType = number;
+
+// Augmenting is necessary so that the TS v6-model.ts compiles properly
+declare module '@balena/sbvr-types' {
+	export interface Types {
+		'Numeric Big Integer': TsTypes;
+	}
+}
+
+// @ts-expect-error we are augmenting SbvrTypes w/ Numeric Big Integer
+SbvrTypes.default['Numeric Big Integer'] = {
+	...SbvrTypes.default['Big Integer'],
+
+	fetchProcessing(data) {
+		if (data == null || typeof data === 'number') {
+			return data;
+		}
+		if (typeof data === 'string') {
+			return parseInt(data, 10);
+		}
+		throw new Error(
+			'Fetched Numeric Big Integer value is not valid: ' + typeof data,
+		);
+	},
```

Comment:
I would lean towards simplifying this function by reusing as much of the `Big Integer` processing as possible:
```suggestion
	fetchProcessing(data) {
		const processedData = SbvrTypes['Big Integer'].fetchProcessing(data);
		return parseInt(processedData, 10);
	},
```
Performance should be equivalent since the `Big Integer` processing should get a string and then directly return it for us but it means less custom code/clearer intention (ie "big integer, but as a number")